README.md

```diff
@@ -18,17 +18,14 @@ Simple example:
 
 
 const templater = require('anytv-templater');
+const i18n = require('anytv-i18n');
```

Comment:
this change is in respect for the v.1.0.0 of anytv-templater. not for the change in i18n.

https://github.com/anyTV/anytv-templater/blob/master/CHANGELOG.md#v100